.github/workflows/cache_data.yaml

```
@@ -34,7 +34,7 @@ jobs:
       # Install GMT and other required dependencies from conda-forge
       - name: Install dependencies
         run: |
-          conda install conda-forge/label/dev::gmt=6.2.0rc2 \
+          conda install -c conda-forge gmt=6.2.0 \
```

`-c conda-forge` is unnecessary because the `setup-miniconda` step already add the conda-forge channel, right?
```suggestion
          conda install gmt=6.2.0 \
```